go-packages/meep-net-char-mgr/net-char-mgr.go

```diff
@@ -59,6 +59,17 @@ type NetChar struct {
 	Jitter     float64
 	PacketLoss float64
 	Throughput float64
+	Distribution string
+}
+
+// NetChar
+type ElemNetChar struct {
+        Latency    float64
+        Jitter     float64
```

Comment:
This is inconsistent.
LatencyDistribution is used everywhere in the model, frontend, etc. and here we switch to Jitter

Reply:
yes... jitter is used mainly in the net-char-package instead of latencyVariation. I agree this might bring confusion. Should this one be moved to latency variation or model should be changed to jitter first?